lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -349,16 +352,21 @@ def test_results_move_relay(monkeypatch, caplog, setup):
 
         runner = CliRunner(mix_stderr=False)
 
+        arg_list = [
+            TestResultsMove.CTRL_SWITCH,
+            TestResultsMove.CTRL_TEXT,
+            TestResultsMove.DELN_SWITCH,
+            TestResultsMove.RELAY_SWITCH,
+            TestResultsMove.RELAY_TEXT,
+        ]
+
+        if brief:
+            arg_list.append(TestResultsMove.BRIEF_SWITCH)
+
         # Test --no-delete
         result = runner.invoke(
             main,
-            args=[
-                TestResultsMove.CTRL_SWITCH,
-                TestResultsMove.CTRL_TEXT,
-                TestResultsMove.DELN_SWITCH,
-                TestResultsMove.RELAY_SWITCH,
-                TestResultsMove.RELAY_TEXT,
-            ],
+            args=arg_list,
         )
```

Comment:
Unless you prefer the multi-line format, I suggest that you remove the trailing comma -- without it, I believe that Black will put this all on one line (as it did in `lib/pbench/test/unit/agent/task/test_results_push.py`).